ctx.go

```diff
@@ -444,6 +426,49 @@ func (c *DefaultCtx) Cookie(cookie *Cookie) {
 	fasthttp.ReleaseCookie(fcookie)
 }
 
+// DetachContext creates a derived independent context that can be used in a new goroutine that
+// is designed to outlive the request-response cycle. copykeys represents which keys from
```

Comment:
This goes against what Fiber stands for, which is performance.

Reply:
1. The rationale for the previous Context()/SetContext() was for passing a context to a goroutine (presumably) that was meant to outlive the request-response cycle.
2.  Values **need** to be copied if you are going to spawn a goroutine and use values. There is no choice. In fact, this function makes it clear to the user what is happening behind the scenes. 
3. Previously, if you were going to spawn a new go routine, it kept a reference.

Reply:
We can figure something out later in a proposal. The implementations needs to be done without external deps.